src/test/HostActivationTests/GivenThatICareAboutMultilevelSharedFxLookup.cs

```diff
@@ -354,6 +382,170 @@ public void Roll_Forward_On_No_Candidate_Fx_Fails_If_No_Higher_Version_Is_Availa
                 .HaveStdOutContaining("9999.1.0");
         }
 
+        [Fact]
+        public void Multiple_SharedFxLookup_Independent_Roll_Forward()
+        {
+            var fixture = PreviouslyBuiltAndRestoredPortableTestProjectFixture
+                .Copy();
+
+            var dotnet = fixture.BuiltDotnet;
+            var appDll = fixture.TestProject.AppDll;
+
+            string runtimeConfig = Path.Combine(fixture.TestProject.OutputDirectory, "SharedFxLookupPortableApp.runtimeconfig.json");
+            SetRuntimeConfigJson(runtimeConfig, "7777.0.0", null, useUberFramework: true);
+
+            // Add versions in the exe folders
+            AddAvailableSharedFxVersions(_exeSharedFxBaseDir, "9999.0.0");
```

Comment:
Do we need/want to delete these things out when the tests finish?

Reply:
I think that is a good practice, but we are inconsistent currently. I'll take a look.

Reply:
Add deletion, including to pre-existing code